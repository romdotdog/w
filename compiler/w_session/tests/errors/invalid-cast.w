fn main() {
    let foo = <*i32>1;
}