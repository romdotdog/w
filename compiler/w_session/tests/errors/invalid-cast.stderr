tests/errors/invalid-cast.w:2:15: error: check your cast or use reinterpret `<type!>x;`
    let foo = <*i32>1;
              ^^^^^^
